src/FortuneWindow.cpp

```diff
@@ -64,24 +53,28 @@ FortuneWindow::FortuneWindow(void)
 		fFortune.LastFilename(&title);
 		title.Prepend("Fortune: ");
 		SetTitle(title.String());
-		
+
 		fTextView->SetText(fortune.String());
 	}
-	else	
+	else
 	{
 		fTextView->SetText("Fortuna had a problem getting a fortune.\n\n"
 			"Please make sure that you have installed fortune files to "
 			"the folder ");
 		fTextView->Insert(path.Path());
 		fTextView->Insert(".");
 	}
-	
+
 	next->MakeFocus(true);
-	
-	// Calculate a good width for the window
-	SetSizeLimits(45 + close->Bounds().Width() + next->Bounds().Width(),30000,200,30000 ) ;
-	r = BScreen().Frame();
-	MoveTo((r.Width()-Bounds().Width())/2,r.Height() / 4);
+	next->MakeDefault(true);
+
+	BLayoutBuilder::Group<>(this, B_VERTICAL, B_USE_DEFAULT_SPACING)
+		.SetInsets(B_USE_WINDOW_INSETS)
+		.Add(sv)
+		.AddGroup(B_HORIZONTAL)
+			.Add(next)
```

Comment:
```suggestion
			.AddGlue()
			.Add(next)
```

Reply:
I didn't add AddGlue because I didn't personally like how the result of that looked from my point of view, but I'll commit this anyways.

Reply:
> I didn't add AddGlue because I didn't personally like how the result of that looked from my point of view, but I'll commit this anyways.

You can't resize the window horizontally without a glue.